src/xinterpreter.cpp

```diff
@@ -131,6 +131,42 @@ namespace xeus
         }
     }
 
+    nl::json xinterpreter::publish_execution_output(int execution_count,
+                                                const std::string& status,
+                                                const std::string& ename,
+                                                const std::string& evalue,
+                                                const std::string& data_type,
+                                                const std::vector<std::string>& data_content,
+                                                const std::vector<std::string>& trace_back)
+    {
+        if (m_publisher)
```

Comment:
This check is not needed

Reply:
Thanks Martin :)
I also had a chat with Johan about it